custom_components/powercalc/data/signify/1740193P0/model.json

```
@@ -0,0 +1,16 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plus 1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.6.6:docker"
+    },
+    "aliases": [
+        "Hue outdoor wall"
```

I see this light marketed as Lucca when searching for it. Would be nice to also have that reflected in the listing.

```suggestion
        "Hue Outdoor Lucca"
```